lessons/Algebra2/logarithmic3-changing-scale/langs/en-us/index.adoc

```diff
@@ -169,6 +168,7 @@ Students will be switching the x-axis of their graph from linear to logarithmic
 @slidebreak
 
 @QandA{
+Before we discuss the work you just did, let's take a minute to think about how treadmills work.
```

Comment:
@schanzer. Added this because there was a bridge missing. Think this is enough? Or do you have other ideas?